src/main/java/org/cryptomator/ui/convertvault/ConvertVaultModule.java

```diff
@@ -120,7 +122,7 @@ static FxController provideNewPasswordController(ResourceBundle resourceBundle,
 	@IntoMap
 	@FxControllerKey(RecoveryKeyValidateController.class)
 	static FxController bindRecoveryKeyValidateController(@ConvertVaultWindow Vault vault, @ConvertVaultWindow VaultConfig.UnverifiedVaultConfig vaultConfig, @ConvertVaultWindow StringProperty recoveryKey, RecoveryKeyFactory recoveryKeyFactory) {
-		return new RecoveryKeyValidateController(vault, vaultConfig, recoveryKey, recoveryKeyFactory);
```

Comment:
doe we need a property for the recvoeryActiontype? If the value is not obseved/listened to changes or we have to change it, then remove the property wrapper